Diagnostics/ManagedAvailabilityTroubleshooter/ManagedAvailabilityTroubleshooter.ps1

```diff
@@ -553,7 +577,7 @@ function CheckifthiscanbeaknownissueusingMonitor {
     if ($Script:lastProbeerror) {
         if ($Script:lastProbeerror.Exception -like "*The underlying connection was closed*") {
             Write-Host -foreground yellow "This probe error message related to underlying connection closed has been seen when connection for loopback adapter has been blocked at lower level before reaching Exchange`n"
-            Write-Host -foreground yellow "You can check in IIS Default Web Site /Actions pane / Bindings that “All Unassigned” is used and this has not been changed to only allow specific IP.`n"
+            Write-Host -foreground yellow "You can check in IIS Default Web Site /Actions pane / Bindings that `“All Unassigned`” is used and this has not been changed to only allow specific IP.`n"
```

Comment:
Can we replace these curly quotes with normal quotes? Seems like an encoding issue waiting to happen.